aiogram/dispatcher/filters/builtin.py

```diff
@@ -66,8 +66,19 @@ def __init__(self, commands: Union[Iterable, str],
                 @dp.message_handler(commands=['myCommand'], commands_ignore_caption=False, content_types=ContentType.ANY)
                 @dp.message_handler(Command(['myCommand'], ignore_caption=False), content_types=[ContentType.TEXT, ContentType.DOCUMENT])
         """
-        if isinstance(commands, str):
+        if isinstance(commands, (str, BotCommand)):
             commands = (commands,)
+        elif isinstance(commands, Iterable):
+            if not all(map(lambda cmd: isinstance(cmd, str), commands)):
```

Comment:
- When `commands` contains `BotCommand`, the check will fail.
- Replacing `map` with generator expression would be more readable, IMO.

```suggestion
            if not all(isinstance(cmd, (str, BotCommand)) for cmd in commands):
```

Reply:
test was failed and i didn't notice🙈